### ISSUE TITLE
Disabled menu items not setting aria-disabled

### ISSUE THREAD

**Issue by dlockhart** [BrightspaceUI/core#718]
When disabling a `<d2l-menu-item disabled>`, while visually and functionally the item acts disabled, `aria-disabled="true"` isn't getting applied to the element with the `menuitem` role. So to screen readers, it does not announce that the item is disabled.

Sample resolved markup:

```html
<d2l-menu-item disabled class="d2l-contextmenu-item" text="Paste" role="menuitem" tabindex="-1">
    <!-- shadow root --->
</d2l-menu-item>
```
